src/core/auth/components/loginform.css

```diff
@@ -0,0 +1,111 @@
+h1 {
```

Comment:
lets keep the name casing of css files same as for the component names, like LoginForm.css

Reply:
Please avoid changing styles of the global tags directly from the components level side. In this case, the component changes style of H1 tag for the entire application. If we really want that then we should use global styles.css. Otherwise, we should be more specific and add some classes to the tag to isolate its styles from any other

Reply:
I will wrap the title in div to make sure no global changes to such tags as H1